packages/paste-core/components/breadcrumb/src/index.tsx

```diff
@@ -46,11 +45,12 @@ const BreadcrumbItem = React.forwardRef<HTMLAnchorElement, BreadcrumbItemProps>(
         lineHeight="lineHeight20"
       >
         {href ? (
-          <Anchor element={`${elementName}_ANCHOR`} href={href} ref={ref}>
+          <Anchor {...safelySpreadBoxProps(props)} element={`${elementName}_ANCHOR`} href={href} ref={ref}>
```

Comment:
As a note worth mentioning, anywhere you are using one of our own components, you _should_ in _theory_ be able to skip using `safelySpreadBoxProps`, as the component you are using, `Anchor` in this case, is already doing it at it's level. 

We therefore don't need to use it twice to render the same DOM element.

Good rule of thumb, use `safelySpreadBoxProps` on a `<Box />` only

Reply:
o0o0o a good note, thanks @SiTaggart! I'd like to change it to`{...props}` for consistency, but when I try to do that, I get some typing errors. I'll see if anyone's available to help me with this.

Reply:
Done!